src/options.html

```diff
@@ -2,7 +2,7 @@
 <html>
 
 <head>
-    <title>BM Refresher</title>
+    <title>DMV Appointment Checker - Options</title>
```

Comment:
Not sure why this didn't get picked up. My fork has [this line](https://github.com/ssgao/dmv-appointement-checker/blob/master/src/options.html#L6)
```
<meta http-equiv="content-type" content="text/html; charset=utf-8">
```
Without this, I'm seeing gibberish in my browser.

Reply:
Okay added that line. Thanks